README.md

```diff
@@ -297,6 +297,10 @@ The Azure MCP Server supercharges your agents with Azure context. Here are some
 
 * Get the Bicep schema for specific Azure resource types
 
+### 🏗️ Cloud Architect
+
+* Design Azure cloud architectures through guided questions
```

Comment:
Use this from the tool description instead?

Azure architecture design tool that gathers requirements through guided questions and recommends optimal solutions.